mypyc/test-data/irbuild-basic.test

```diff
@@ -1358,7 +1358,7 @@ def lst(x):
 L0:
     r0 = len x :: list
     r1 = 0
-    r2 = CPyTagged_IsNe(r0, r1)
+    r2 = r0 != r1
```

Comment:
This is caused by https://github.com/python/mypy/commit/358522e28cd58d95daf36256c46eb7ffcc55eea4